backend/server.ts

```diff
@@ -1,9 +1,29 @@
-const express = require("express");
+import express, { Request, Response } from "express";
 const app = express();
+const mongoose = require("mongoose");
+const user_connection_url = "mongodb+srv://naacpDev:hack4impact2022@cluster0.dbyhj.mongodb.net/userDB?retryWrites=true&w=majority";
```

Comment:
I also realized the URL is here... Should I just replace this with placeholder text?